docs/docs/documentation/deployment/debugging.md

```diff
@@ -133,7 +133,31 @@ By sending a request and observing the response, you can verify an agent's behav
 
 ## System Logs
 
-System logs provide additional insights into the system's behavior. These logs are written to a log file at the root of the project directory.
+System logs provide detailed insights into the system's behavior. The logging is configured in the `configs/logging_config.ini` file, which controls both console (STDOUT) and file-based logging.
```

Comment:
Suggestion: 
From: The logging is configured
To: The logging behavior is configured